src/panels/lovelace/cards/hui-gauge-card.ts

```diff
@@ -133,8 +133,12 @@ class HuiGaugeCard extends LitElement implements LovelaceCard {
             .unit_of_measurement ||
           ""}
           style=${styleMap({
-            "--gauge-color": this._computeSeverity(entityState),
+            "--gauge-color": this._config.dial
+              ? "var(--label-badge-blue)"
+              : this._computeSeverity(entityState),
           })}
+          ?dial=${this._config!.dial}
+          .levels=${this._config!.dial ? this._severityLevels() : []}
```

Comment:
```suggestion
          .levels=${this._config!.dial ? this._severityLevels() : undefined}
```

Reply:
We should not create a new array in every render, as it will trigger the gauge to completely re-render